dockers/hpu-ci-runner/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG DIST="latest"
+ARG VERSION="1.4.0"
+ARG PYTORCH_INSTALLER_VERSION="pytorch-installer-1.10.2"
+
+FROM vault.habana.ai/gaudi-docker/${VERSION}/ubuntu20.04/habanalabs/${PYTORCH_INSTALLER_VERSION}:${DIST}
+
+LABEL maintainer="https://vault.habana.ai/"
+
+# Remove the pytorch-lightning available in the docker image
+RUN pip uninstall pytorch-lightning -y
```

Comment:
noob question: why remove it?

Reply:
This docker image has a pytorch lightning version already and the Pytorch team want to uninstall it and re-install as part of the CI workflow. 
@jerome-habana  Please, do confirm if this is expected.